service/tbc/tbc.go

```diff
@@ -67,19 +63,22 @@ const (
 var (
 	zeroHash = new(chainhash.Hash) // used to check if a hash is invalid
 
+	localnetSeeds = []string{
+		"bitcoind:18444",
```

Comment:
What do you think about defaulting this to `127.0.0.1:18444` and overriding it in the Docker environment when `bitcoind` will exist?

Reply:
that's a good question @joshuasing ; I was on the fence about this 🤔 do you like `127.0.0.1` better?

Reply:
Agreement to default to `127.0.0.1` was made on a separate platform, marking as resolved.